front_end_pocket_poll/lib/events_widgets/events_list.dart

```diff
@@ -149,8 +179,12 @@ class _EventsListState extends State<EventsList> {
             ));
       }
 
+      this.loadingBatch = false;
+
       return Scrollbar(
+          key: UniqueKey(),
```

Comment:
Why was the scrollbar given a key?

Reply:
It makes it reload visually when it rebuilds since the widget tree will have different keys than before.